src/js/components/List/List.js

```diff
@@ -5,21 +5,17 @@ import { Box } from '../Box';
 import { InfiniteScroll } from '../InfiniteScroll';
 import { Keyboard } from '../Keyboard';
 import { Text } from '../Text';
-import { focusStyle, genericStyles } from '../../utils';
+import { focusStyle, genericStyles, useForwardedRef } from '../../utils';
 
 const StyledList = styled.ul`
   list-style: none;
   ${props => !props.margin && 'margin: 0;'}
   padding: 0;
   ${genericStyles}
 
-  ${props =>
-    props.onClickItem &&
-    `
-    &:focus {
-      ${focusStyle({ forceOutline: true, skipSvgChildren: true })}
-    }
-  `}
+  &:focus {
```

Comment:
Can we put `&:focus` inside the check for `tabIndex`? That will keep the CSS a bit cleaner.

Reply:
When I place the &:focus instead of the check, it does not apply the focus style. This is the code I was using:
```
  ${props =>
    props.tabIndex >= 0 &&
    `
    &:focus {
      ${focusStyle({ forceOutline: true, skipSvgChildren: true })}
    }
  `}
```

Reply:
I don't believe any CSS is being added for List without an onClickItem because the browser is not interpreting it as focusable